dhis-2/pom.xml

```diff
@@ -2324,6 +2326,11 @@
         <artifactId>tree</artifactId>
         <version>${tree.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.testcontainers</groupId>
+        <artifactId>junit-jupiter</artifactId>
```

Comment:
does it have scope test? if not we should set it 😄 maybe even be explicit about it even it if already is

Reply:
This is not a test dependency, it is used in `IntegrationTestConfig` that is not in the test folder.

Reply:
Should we set it to test here and override it to compile for the `dhis-support-test` module? Just to make sure anyone that would use it directly does not pull it in as a compile time dependency?

Reply:
I think we have the `maven-dependency-plugin` that is helping us with it.
If someone is using it directly in a test and add it as a compile time dependency will get an error from the analyser